src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -256,5 +256,7 @@
   "virtualAgentEnableMsftPvaTesting": "virtual_agent_enable_msft_pva_testing",
   "virtualAgentEnableNluPvaTesting": "virtual_agent_enable_nlu_pva_testing",
   "yellowRibbonAutomatedDateOnSchoolSearch": "yellow_ribbon_automated_date_on_school_search",
-  "toggleVyeAdressDirectDepositForms": "toggle_vye_adress_direct_deposit_forms"
+  "toggleVyeAdressDirectDepositForms": "toggle_vye_adress_direct_deposit_forms",
```

Comment:
'toggle_vye_adress_direct_deposit_forms' also has the typo for `address`. Is this in use already by your team?

Reply:
yeah I can change that in other PR for my team. I have changed the one which is used for the profile

Reply:
I have updated in this pr https://github.com/department-of-veterans-affairs/vets-website/pull/30218